### PR TITLE
Improved deal screen mod addon

### PR DESCRIPTION
**Pull request by ricanuck**
As promised. I did my best to incorporate all the CQUI elements and avoid a regression. It should be fine, but double check.

More importantly, test test test! This is a great addon, and I am 110% sure it will stay as a key part of CQUI. I corrected some bugs/issues present in the original. Credits to mironos for the original mod in Steam workshop.

### CODE REVIEW

**Comment by chaorace** [comment]
The good news is that this nightly will be relatively isolated from the workshop release for the next week. I feel merging now (after a round of internal testing, of course!) is the best decision here in order to catch any bugs that arise

File: Assets/UI/diplomacydealview.lua [hunk 1]
```diff
@@ -1,2434 +1,2769 @@
--- ===========================================================================
```
Comment (chaorace) [inline Assets/UI/diplomacydealview.lua]: What's up with the diff for this file? Something isn't quite right here

File: cqui.modinfo [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
+    <Authors>vans163, chaorace, zgavin, olegbl, jacks0n, OfekA, [NEWLINE]Proustldee, RatchetJ, kblease, benjaminjackman, LordYanaek, [NEWLINE]zeyangl, velit, the-m4a, Frozen-In-Ice, Remolten, bestekov, [NEWLINE]bolbass, SpaceOgre, cpinter, JHCD, paavohuhtala, [NEWLINE]Aristos</Authors>
     <SpecialThanks>vans163 for his original qui mod. [NEWLINE]astog for his Better Trade Screen mod + More Lenses mod. [NEWLINE]Ace for his Next City Plot mod. [NEWLINE]Divine Yuri for his Custom City Panel mod. [NEWLINE]Greg Miller for his Unit Report Screen mod. [NEWLINE]Lozenged/kblease for his Production Queue mod. [NEWLINE]ZhouYzzz for providing the Chinese translation of CQUI. [NEWLINE]Simone1974 for providing the Italian translation of CQUI. [NEWLINE]e1ectron for providing the Russian translation of CQUI. [NEWLINE]sejbr for providing the Polish translation of CQUI. [NEWLINE]frytom/maxap/JHCD for providing the German translation of CQUI. [NEWLINE]lctrs for providing a partial French translation of CQUI. [NEWLINE]wbqd for providing the Korean translation of CQUI.</SpecialThanks>
```
Comment (chaorace) [inline cqui.modinfo]: The original mod author needs to be credited here, just after the Lozenged credit and just before the LOC credits start

File: Assets/UI/diplomacydealview.lua (hunk 1 above)
Comment (ricanuck) [inline Assets/UI/diplomacydealview.lua]: A lot of new code, but yes, it caught my attention too, it should have shown the diffs only... maybe there is a threshold in lines above which it shows as diff? File looks fine and works well in my setup.

File: cqui.modinfo (hunk 2 above)
Comment (ricanuck) [inline cqui.modinfo]: Done.

File: Assets/UI/diplomacydealview.lua (hunk 1 above)
Comment (JHCD) [inline Assets/UI/diplomacydealview.lua]: Looks like change in linebreak oder spaces

**Comment by chaorace** [comment]
Well this has been sitting long enough, I'm merging this PR now

**Pull request merged**